### ISSUE TITLE
avito

### ISSUE THREAD

**Issue by advokat11** [mtxadmin/ublock#57]
Не работает кнопка "найти" на авито, с помощью которой можно искать объявления по текстовому запросу.

**Comment by advokat11** [comment]
Также не работают сообщения и не показывается кнопка с номером автора объяления.

**Comment by mtxadmin** [comment]
Да, спасибо, там скрипты поломались. Сейчас вроде заработало?

**Comment by advokat11** [comment]
Нет, сообщения пользователей это теперь невидимые панельки на которые можно нажать, но непонятно что это за диалог. Также в самой переписке нет аватарки того, с кем переписываешься.
